config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  # assigning temp root to the drivers page, replace with the homepage once its set up
+  root 'homepage#index'
+
+  resources :homepage, only: [:index]
+
+  resources :trips
+
+  resources :drivers do
+    resources :trips, only: [:index]
```

Comment:
Why do we need to nest a trips route into drivers? This gives us the route `/drivers/:id/trips`... which we don't have a requirement to have